cumulusci/cli/cci.py

```diff
@@ -223,9 +227,12 @@ def main(args=None):
             show_debug_info() if debug else click.echo("\nAborted!")
             sys.exit(1)
         except Exception as e:
-            show_debug_info() if debug else handle_exception(
-                e, is_error_command, tempfile_path
-            )
+            if debug:
+                show_debug_info()
+            elif should_show_exceptions and not isinstance(e, UsageErrors):
+                raise
+            else:
+                handle_exception(e, is_error_command, tempfile_path)
```

Comment:
I think Brandon's intention was that `handle_exception` contains the logic for deciding what to display after an exception. Let's put the new conditional in there.

Reply:
d3b626236dff6c3f40a2e13b9f5115d46607d21a